src/compiler/parser.ts

```diff
@@ -6526,6 +6526,8 @@ namespace ts {
                     const result = target === PropertyLikeParse.Parameter ?
                         <JSDocParameterTag>createNode(SyntaxKind.JSDocParameterTag, atToken.pos) :
                         <JSDocPropertyTag>createNode(SyntaxKind.JSDocPropertyTag, atToken.pos);
+                    let comment: string | undefined;
+                    if (indent !== undefined) comment = parseTagComments(indent + scanner.getStartPos() - atToken.pos);

```

Comment:
If only nested `@param`s were missing comments before this change, where was the comment parsed for non-nested ones? Does this change now parse the comments for all `@param`s here?

Reply:
Yeah, previously it was always done at the bottom of `parseTag`. But that didn't work if the comment had already been skipped.